sdk/src/jmh/java/io/opentelemetry/sdk/metrics/TestSdk.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.sdk.metrics;
+
+import com.google.errorprone.annotations.Immutable;
+import io.opentelemetry.metrics.DefaultMeter;
+import io.opentelemetry.metrics.Meter;
+import io.opentelemetry.sdk.common.InstrumentationLibraryInfo;
+import io.opentelemetry.sdk.internal.MillisClock;
+import io.opentelemetry.sdk.resources.Resource;
+
+public enum TestSdk {
+  NO_SDK(
```

Comment:
API_ONLY? I don't know maybe you can think of a better name :), kind of don't like `NO_SDK`.

Reply:
I actually really like API_ONLY- I think that's both clearer and more positive.